### PR TITLE
Add 4.0.0 changelog

### PR DESCRIPTION
**Pull request by floren**

### CODE REVIEW

File: changelog/4.0.0.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+## Ingesters & Ingest Library Changes
+* Implemented new, more efficient cache system
+* Added preprocessor to handle Amazon VPC flow logs
+* Fixed bug where cached entries could fail tag translation on ingester restart
+* Kinesis ingester now reports metrics via `gravwell` tag
+* Kafka Federator now supports TLS-encrypted Kafka transports
+* Fixed line number reporting on ingester log messages
+* Improved logging in early ingester startup
+* Added code to handle Zeek's tendency to leak PIDs
+
```
Comment (traetox) [inline changelog/4.0.0.md:56]: we also enabled transparent compression in the ingest transport.

Comment (traetox) [inline changelog/4.0.0.md:56]: Also added a new gravwellforward preprocessor that can duplicate entries out to other Gravwell indexers.

File: changelog/4.0.0.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+## Kits
+* Network Enrichment: Provides databases for enriching network data (geoip, ASN, IP protocols, etc.)
+* CoreDNS: Analyze entries from the [Gravwell CoreDNS plugin](https://github.com/gravwell/coredns)
+* Netflow v5: Analyze Netflow v5 flow records
+* IPFIX: Analyze IPFIX flow records
+* Grok: Pre-built database of extraction patterns for use with the [grok module](https://docs.gravwell.io/docs/#!search/grok/grok.md)
+
```
Comment (traetox) [inline changelog/4.0.0.md:63]: go ahead and call out the following additional kits:
* zeek
* Windows
* syslog

They will be ready for launch or shortly thereafter

File: changelog/4.0.0.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+## General/Miscellaneous
+* Added Debian and RPM packages for datastore and loadbalancer components
+* Migrated some personally-maintained code forks into github.com/gravwell umbrella account
+* Transitioned most Gravwell open-source code to a monorepo: [github.com/gravwell/gravwell](https://github.com/gravwell/gravwell)
+* Community Edition licenses will no longer allow the creation of more than 2 users (any existing users will be preserved)
```
Comment (traetox) [inline changelog/4.0.0.md:68]: * Fixed issue in zeek docker container that was causing process PID leakage

File: changelog/4.0.0.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+* Fixed off-by-one error in query tag handling which could cause crashes
+* Fixed bug in macro creation in which groups were not properly preserved
+* Fixed SSO bug where user removal from a group in Active Directory was not mirrored in Gravwell
+* Fixed incorrect nil pointer handling in scheduled search `httpPost` function
+* Fixed potential webserver crash during shutdown due to lingering websockets
+* Fixed bug where `gauge` module could not parse multiple `(magnitude label)` specifications
+* Fixed bug in `gauge` module where enumerated values containing underscores were not parsed correctly
+
+## Ingesters & Ingest Library Changes
+* Implemented new, more efficient cache system
```
Comment (djfritz) [inline changelog/4.0.0.md:48]: Added new cache mode to allow ingesting even without an indexer connection.

**Pull request merged**